targets/targets.json

```diff
@@ -1345,6 +1345,23 @@
         "release_versions": ["2", "5"],
         "device_name": "STM32L432KC"
     },
+    "NUCLEO_L433RC": {
+        "inherits": ["FAMILY_STM32"],
+        "supported_form_factors": ["ARDUINO"],
+        "core": "Cortex-M4F",
+        "extra_labels_add": ["STM32L4", "STM32L433xC", "STM32L433RC"],
+        "config": {
+            "clock_source": {
+                "help": "Mask value : USE_PLL_HSE_EXTC (need HW patch) | USE_PLL_HSE_XTAL (need HW patch) | USE_PLL_HSI | USE_PLL_MSI",
+                "value": "USE_PLL_MSI",
+                "macro_name": "CLOCK_SOURCE"
+            }
+        },
+        "detect_code": ["0770"],
```

Comment:
I have the ID `0779 `when I connect my NUCLEO-L433RC-P.

Can you confirm the ID you see in the MBED.HTM file (first 4 digits in the code field) ?

`<meta http-equiv="refresh" content="0; url=http://mbed.org/device/?code=07790221172C637A2868F53E"/> `

Reply:
Yes my Code is: code=07790221172F6363286BF527